app-arch/innoextract/innoextract-1.8.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="A tool to unpack installers created by Inno Setup"
+HOMEPAGE="http://constexpr.org/innoextract/"
+SRC_URI="http://constexpr.org/innoextract/files/${P}.tar.gz"
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug +iconv +lzma"
+
+RDEPEND="
+	dev-libs/boost:=
+	iconv? ( virtual/libiconv )
+	lzma? ( app-arch/xz-utils )"
+DEPEND="${RDEPEND}"
+
+DOCS=( README.md CHANGELOG )
```

Comment:
These are part of default `einstalldocs` array, so you can remove this part and they will still be installed.